lib/polo/adapters/postgres.rb

```diff
@@ -0,0 +1,19 @@
+module Polo
+  module Adapters
+    module Postgres
+      def on_duplicate_key_update(inserts, records)
+        raise 'Not implemented for Postgres'
+      end
+
+      def ignore_transform(inserts, records)
+        insert_and_record = inserts.zip(records)
```

Comment:
I like this implementation, but find it a little hard to follow because of all the gsubing and the regexes.
Can you write some high level comments for this? (in the same format as our other comments) and also include some input and ouput samples?